lib/manageiq/api/common/graphql.rb

```diff
@@ -80,6 +80,17 @@ def self.openapi_graphql_description
           }
         end
 
+        def self.ordered_search(scope, _args)
+          scope.order(:id)
+        end
+
+        def self.paged_search(scope, args)
```

Comment:
Any way we can share some of this code? https://github.com/ManageIQ/manageiq-api-common/blob/master/lib/manageiq/api/common/paginated_response.rb#L7

Reply:
No problem, was able to update GraphQL to actually leverage that class by exposing a helper method in there to expose the updated search scope. https://github.com/ManageIQ/manageiq-api-common/pull/78/commits/e1fcac7aabe7f7adb2c56238c2bd64e2e08e870f

so we now get the default common order, offset and limit set by that class.